pkg/cloudprovider/provider/vsphere/provider.go

```diff
@@ -277,6 +277,10 @@ func (p *provider) Validate(spec v1alpha1.MachineSpec) error {
 		return errors.New("specified target network (VMNetName) in cluster, but no source network (TemplateNetName) in machine")
 	}
 
+	if config.CPUs > 8 {
+		return errors.New("number of CPUs could not be greater than 8")
```

Comment:
can you replace `could` with `must not`? Other than that LGTM

Reply:
👍